modules/displayBook.js

```diff
@@ -0,0 +1,17 @@
+export default class Books {
+  constructor(array) {
+    this.bookArray = array;
+  }
+
+  bookObj(bookTitle, authorName) {
+    const eachBook = {
+      title: bookTitle,
+      author: authorName,
+    };
+    this.bookArray.push(eachBook);
+  }
+
+  booksFilter(eachBook) {
+    this.bookArray = this.bookArray.filter((book) => book !== eachBook);
+  }
```

Comment:
- [ ] good job refactoring :clap: , Please notice that one of the main goal of this project is to refactor it to use ES6 suntax, which uses arrow functions instead, I suggest we change all the functions to be arraw functions :pray: 
ex: 
```
const booksFilter = (eachBook) => {
    this.bookArray = this.bookArray.filter((book) => book !== eachBook);
  }
```